packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts

```diff
@@ -69,6 +69,8 @@ export const getAudioReceiverMqa = ({
     (totalFecPacketsDiscarded - lastFecPacketsDiscarded);
   audioReceiver.common.fecPackets = fecRecovered;
```

Comment:
Wasn't there a conversation in the "Ask MQA" space about `fecPackets` being incorrect here? From Nathan: "`fecPackets` in MQE is the number of FEC packets received, regardless of whether they were used in recovery or not. So this sounds like the WebRTC stats `fecPacketsReceived` field."

Should we fix that here?

Reply:
Ah yeah, good catch @brycetham